sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -242,6 +255,18 @@ private IEnumerable<Instruction> BuildNewInstance(ExpressionSyntax expressionSyn
             return new[] { newObjectInstruction };
         }
 
+        private IEnumerable<Instruction> BuildNewArrayInstance(ExpressionSyntax syntaxNode,
+            IArrayTypeSymbol arrayTypeSymbol,
+            InitializerExpressionSyntax arrayInitializationNode)
```

Comment:
[cosmetic] It would be better to keep the current class style (i.e. to not put all new parameters on new line but to simply wrap instead).